modules/apps/object/object-test/src/testFunctional/tests/Object.testcase

```diff
@@ -510,6 +510,73 @@ definition {
 		}
 	}
 
+	@description = "Verify it is possible to add an Object Entry Title Field when changing the localization on Instance Settings"
+	@priority = 4
+	test CanViewTitleFieldWhenChangeLocalization {
+		task ("Given: A default language in the virtual instance changed") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Configuration",
+				panel = "Control Panel",
+				portlet = "Instance Settings");
+
+			SystemSettings.gotoConfiguration(
+				configurationCategory = "Localization",
+				configurationName = "Language",
+				configurationScope = "Virtual Instance Scope");
+
+			PortalSettings.configureCurrentLanguagesCP(defaultPortalLanguage = "Portuguese (Brazil)");
+		}
+
+		task ("And given: An Account and a Custom Object with a Relationship 1toM of Account to Custom Object") {
+			JSONAccountEntry.addAccountEntry(
+				accountEntryName = "Accounts Name",
+				accountEntryType = "Business");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectAdmin.addNewObject(
+				fieldLabelName = "Custom Object 178761",
+				pluralLabelName = "Custom Objects 178761");
+
+			CreateObject.saveObject();
```

Comment:
Could you replace these steps to do via API?

```
ObjectAdmin.addObjectViaAPI(
	labelName = "Custom Object 178761",
	objectName = "CustomObject178761",
	pluralLabelName = "Custom Objects 178761");
```

Reply:
For some reason the language change in the virtual instance prevented an object from being created by the API, so I need to do it through the UI.